Gemfile

```diff
@@ -1,18 +1,17 @@
 source 'https://rubygems.org'
 
-ruby '2.4.1'
+ruby '2.4.2'
 
-gem 'rails', '~> 4.2.0'
+gem 'rails', '~> 5.1.0'
 
 gem 'active_model_serializers'
 gem 'acts-as-taggable-on'
-gem 'carrierwave', '~> 0.10.0'
+gem 'carrierwave'
```

Comment:
I'd recommend keeping the versions explicitly here (might help for future upgrades)

Reply:
done